src/dirs.proj

```diff
@@ -15,6 +15,10 @@
     <Project Include="shims\ApiCompat.proj" />
   </ItemGroup>
 
+  <ItemGroup Condition="'$(DirectoryToBuild)' != ''">
+    <Project Include="$(DirectoryToBuild)/**/*.csproj" />
```

Comment:
I see, so it just builds potentially out of order and inefficiently, got it.  I don't care much since this is just a hook for manual building.  I can see why you used the wildcards.  Personally I'll continue to build individual projects.